tabbycat/results/views.py

```diff
@@ -620,11 +618,10 @@ def get_context_data(self, **kwargs):
         return super().get_context_data(**kwargs)
 
 
-class PrivateUrlBallotScoresheetView(RoundMixin, SingleObjectByRandomisedUrlMixin, BasePublicBallotScoresheetsView):
+class PrivateUrlBallotScoresheetView(RoundMixin, BasePublicBallotScoresheetsView, SingleObjectByRandomisedUrlMixin):
```

Comment:
This inheritance order normally isn't best, normally you want all mixins to come ahead of the base view in the MRO. What needed this to make it work?